CONTRIBUTING.md

```diff
@@ -42,85 +39,63 @@ How to contribute if you are a:
 
 - Graphic Designer / Multimedia
 
-- Marketing
-
 - Translator / Copywriter
 
 - Legal Advise & Copyrights Advisors
 
-
 #### Sponsors
-We want to democratise and make this technology available to everyone. However, there are some costs associated for this kind of projects. For example, when we create a website or a mobile app, people will be able to use the service free-of-charge. However, just for keeping the website online operating 24h/7, there will be some costs.
+We want to democratize and make this technology available to everyone. However, there are some costs associated with this kind of project. For example, when we create a website or a mobile app, people will be able to use the service free-of-charge. However, just for keeping the website online operating 24h/7, there will be some operating costs.
```

Comment:
Actually, both "democratise" and "democratize" are correct. The latter is the American spelling whereas here in Australia we use the 's' spelling. It's ok to use the american spelling but need to be sure it's consistent in all documents.

Reply:
Gotcha, yes :)